ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -348,6 +353,17 @@ open class BottomCommandingController: UIViewController {
         }
     }
 
+    public override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+
+        tokenSet.update(fluentTheme)
```

Comment:
Shouldn't the tokenSet be be updated before the view shows up instead to make sure we have the correct theme?